cluster/cluster.yaml

```diff
@@ -4,6 +4,7 @@ Metadata:
   Tags:
     InfrastructureComponent: "true"
     application: "kubernetes"
+    cluster-provider: "{{ .Cluster.Provider }}"
```

Comment:
I left a comment [here](https://github.bus.zalan.do/teapot/kubernetes-on-aws-e2e/pull/176#issuecomment-5721388) about what this is about.